src/modules/savedjobs/savedjobs.controller.ts

```
@@ -0,0 +1,61 @@
+import { Body, Controller, Get, HttpException, HttpStatus, Post, Query, UseGuards } from '@nestjs/common';
+import { ApiBearerAuth, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { SavedJobsService } from '../savedjobs/services/savedjobs.service';
+import { SavedJobsEntity } from '../../database/entities/savedjobs.entity';
+import { CreateSavedJobDto } from '../savedjobs/dtos/create-savedJob-dto';
+import { AuthGuard } from '@nestjs/passport';
+import { SwaggerFindSavedJobs } from 'src/shared/Swagger/decorators/savedjobs/view-savedjobs.swagger.decorator';
+import { SwaggerCreateSavedJobs } from 'src/shared/Swagger/decorators/savedjobs/create-savedjobs.swagger.decorator';
+import { GetAllSavedJobsDto } from './dtos/get-all-savedjobs.dto';
+import { PageOptionsDto } from 'src/shared/pagination';
+import { FindAllSavedJobsService } from './services/find-all-savedjobs.service';
+import { log } from 'console';
```

ESLint

[error] 'log' is defined but never used.

(@typescript-eslint/no-unused-vars)